dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -44,7 +44,7 @@
   - if resource.username.present?
     .field
       = f.label :username, class: "label-bold"
-      = f.object.username
+      = f.text_field :username, autofocus: false, maxlength: 20
```

Comment:
Can we also do a minlength of 5 on the field? The user model has `validates_length_of :username, within: 5..20, allow_blank: true`